test/addapibackend.js

```diff
@@ -92,16 +113,31 @@ test.describe('Add API Backend', function() {
         CommonUtils.fillSignInForm(driver, 'test@test.test', 'testuser');
         CommonUtils.goToDashboard(driver);
         AddAPIBackendUtil.addNewBackend(driver, {
-            backendName: 'NewBackend',
-            description: 'Test Description',
-            hostName: 'google',
-            portNumber: '80',
-            frontendPrefix: '/test',
-            backendPrefix: '/test'
+            apiName: 'NewBackend',
+            apiDescription: 'Test Description',
+            apiURL: 'google.com'
+        });
+        // Get the URL field in form
+        var errorElement = driver.findElement(By.xpath('//*[@id="addApiForm"]/fieldset/div[3]/span'));
+        errorElement.getText().then(function(text){
+            assert.include(text, 'URL must be a valid URL');
         });
-        var errorElement = driver.findElement(By.xpath('//*[@class="s-alert-box-inner"]/p'));
+    });
+    test.it('8.5 should not add/publish api with no unique name', function() {
+        newUser.added = false;
+        CommonUtils.signIn(driver);
+        CommonUtils.fillSignInForm(driver, 'test@test.test', 'testuser');
```

Comment:
I created user 'robot@test.com', 'robottest' on nightly to run tests, maybe we could use that, could you change these accordingly?